internal/hud/view/view.go

```diff
@@ -9,22 +9,55 @@ import (
 
 type ResourceInfoView interface {
 	resourceInfoView()
+	RuntimeLog() string
+	Status() string
 }
 
 type DCResourceInfo struct {
 	ConfigPath string
-	Status     string
+	status     string
 	Log        string
```

Comment:
do you want to make `log` private as well now that it has an accessor method? (i don't actually have an opinion here.)

Reply:
oh, sure...

(fwiw, making status private wasn't about encapsulation as much as allowing it to compile by removing the name collision)